tinygrad/tensor.py

```diff
@@ -303,6 +303,32 @@ def chunk(self, num, dim):
       slice_params[i][dim] = (k, min(self.shape[dim], k+self.shape[dim]//num))
     return [self.slice(p) for p in slice_params]
 
+  @staticmethod
+  def interpolate(x, scale_factor, mode='nearest'):
+    assert mode == 'nearest', 'Only nearest interpolate available'
+    bs, c, py, px = x.shape
+    return x.reshape(bs, c, py, 1, px, 1).expand(bs, c, py, scale_factor, px, scale_factor).reshape(bs, c, py * scale_factor, px * scale_factor)
+
+  @staticmethod
```

Comment:
If things use numpy like this, they don't belong in tensor.py

Reply:
removed from tensor.py

Reply:
These are still in tensor.py

Reply:
missed it.. I've kept interpolate in tensor.py after removing staticmethod, moved sort and topk